evalai/urls.py

```diff
@@ -24,6 +24,9 @@
 
 from accounts.views import ConfirmEmailView
 
+handler404 = 'evalai.views.page_not_found'
+handler500 = 'evalai.views.server_error'
```

Comment:
Can you please change the name of handler500 to `evalai.views.internal_server_error`